taskfiles/js.yml

```diff
@@ -0,0 +1,41 @@
+version: '3'
+
+tasks:
+  js-deps:
```

Comment:
the convention we're following is to add install-related tasks in `taskfiles/dev.yml` . So for example, this would be `dev:install-node`. If we do this, then we can remove the `npm install` command out of here and put it in the `build` task

Reply:
Did that with the exception of moving `npm install` as that is needed to build node itself